architecture.rst

```diff
@@ -39,9 +39,10 @@ as part of the Slurm-core offering:
 * |slurmctld-badge|: Slurm control node (running ``slurmctld``)
 * |slurmrestd-badge|: Slurm REST service (running ``slurmrestd``)
 
-Additionally we include the `Node Health Check (NHC)
+Additionally we require the `Node Health Check (NHC)
```

Comment:
I did not get this, maybe it is a language issue? Is this understanding correct: "require" implies that the person adds NHC somehow to the system, while "include" means the charms do it, as part of the package?

Reply:
Right. It becomes a requirement for slurmd to operate vs. before where we would include the install of nhc.